debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4

```diff
@@ -241,14 +241,30 @@ createTrigger
       routineBody
     ;
 
+withClause
+    : WITH RECURSIVE? commonTableExpressions
+    ;
+commonTableExpressions
+    : cteName (('(' cteColumnName (',' cteColumnName)* ')')?  AS '(' dmlStatement ')')
+      (',' commonTableExpressions)?
+    ;
+
+cteName
```

Comment:
Can you please prepare a PR for upstream? I wonder if they would like to have the rules done this way or they'd prefer to use `uid` directly in the rules above. Ideally we should do it in the same way.

Reply:
@jpechane created the upstream PR. I will wait for any suggestions from them and apply them here also.